File: Test/sources/beginner/HelloWindow.cpp
#include "Application.h"
#include "MainWindow.h"

using namespace d14uikit;

#define DEMO_NAME L"HelloWindow"

int main()
{
    Application app(DEMO_NAME);

    //------------------------------------------- Initialize UI objects.

    MainWindow mwnd(DEMO_NAME);

    //------------------------------------------- Set UI event callbacks.

    return app.run();
}

File: Test/PyBind/scripts/beginner/HelloWindow.py
from D14UIKit import *

DEMO_NAME = 'HelloWindow'

def main():

    app = Application(DEMO_NAME)

    #------------------------------------------- Initialize UI objects.

    mwnd = MainWindow(DEMO_NAME)

    #------------------------------------------- Set UI event callbacks.

    exit(app.run())

if __name__ == '__main__':
    main()
